<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>460</width>
           <height>131</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="scrollLayout"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="statusTip">
         <string>Add a new credential.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/img/newcred.png</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="pinEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>P.I.N.</string>
        </property>
        <property name="statusTip">
         <string>MemType device P.I.N.</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Enter PIN</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string>Free credentials space in device.</string>
        </property>
        <property name="maximum">
         <number>2048</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="format">
         <string>Used Space %p%</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>482</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRead"/>
    <addaction name="actionWrite"/>
    <addaction name="separator"/>
    <addaction name="actionImport_File"/>
    <addaction name="actionExport_File"/>
    <addaction name="separator"/>
    <addaction name="actionSet_PIN"/>
    <addaction name="actionSet_KeyLayout"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRead"/>
   <addaction name="actionWrite"/>
   <addaction name="actionSet_PIN"/>
   <addaction name="actionImport_File"/>
   <addaction name="actionExport_File"/>
   <addaction name="actionSet_KeyLayout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <action name="actionRead">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/read.png</normaloff>:/img/read.png</iconset>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
   <property name="statusTip">
    <string>Read all the credentials from device.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionWrite">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/write.png</normaloff>:/img/write.png</iconset>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
   <property name="statusTip">
    <string>Write all the credentials to device.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSet_PIN">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/pin.png</normaloff>:/img/pin.png</iconset>
   </property>
   <property name="text">
    <string>Set PIN</string>
   </property>
   <property name="statusTip">
    <string>Change the device P.I.N.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionImport_File">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Import File</string>
   </property>
   <property name="statusTip">
    <string>Load credentials from a file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionExport_File">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Export File</string>
   </property>
   <property name="statusTip">
    <string>Save credentials to a file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSet_KeyLayout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/keylayout.png</normaloff>:/img/keylayout.png</iconset>
   </property>
   <property name="text">
    <string>Set KeyLayout</string>
   </property>
   <property name="statusTip">
    <string>Change the language layout in the device.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string>Exit.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
